### ISSUE TITLE
Let me do a proper review that includes many comments

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode-pull-request-github#298]
Currently I think we can only leave one off comments and then request changes/approve the PR. Reviews can be dismissed which dismissed several comments, also I always use the top-level review comment to give an overall statement on the PR/review, having these disjointed is much less organized.

**Comment by jongio** [comment]
This will block me from using the extension.  Grouping comments into a review is my main PR review workflow.

**Comment by cavalos0086** [comment]
Is there any reason why this issue is closed? All other issues that are related to "pending comments" were closed because they all point to this one.

**Comment by kballard** [comment]
I agree. I would love to use this extension, but the inability to group comments into a review is a hard dealbreaker. I'm not going to subject my coworkers to 40 emails when I could send them a single email with 40 comments.

**Comment by suresh-productiv** [comment]
This is definitely something that is blocking me from using this otherwise wonderful plugin. Any idea when this feature will get picked up?
